### ISSUE TITLE
Incorrect travel stipend status

### ISSUE THREAD

**Issue by Lereena** [outreachy/website#342]
If I understand everything right, Outreachy's travel stipends are valid for two years. 
On the page https://www.outreachy.org/travel-stipend/ one can see the status of the different years participants' stipends. But now all the stipends marked as expired: 
![image](https://user-images.githubusercontent.com/23419187/63380502-639d9980-c39f-11e9-909e-7df96276dc8a.png)
And if rely on information about the duration of validity of the stipend, it turns out that info about dates on the intern's dashboard is incorrect too:
![image](https://user-images.githubusercontent.com/23419187/63380749-da3a9700-c39f-11e9-9c88-f485ba6905d1.png)

**Comment by sagesharp** [comment]
You're right that this is a bug. Thanks for reporting it! I'll take a look into it.
